src/content/zh-TW/2020/performance.md

```diff
@@ -0,0 +1,387 @@
+---
+part_number: II
+chapter_number: 9
+title: 效能
+description: 2020 Web Almanac 的效能章節涵蓋了 <span lang="en">Core WebVitals</span>、<span lang="en">Lighthouse</span> <span lang="en">Performance Score</span>、<span lang="en">First Contentful Paint</span> (FCP) 以及 <span lang="en">Time to First Byte</span> (TTFB)。
+authors: [thefoxis]
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia, exterkamp]
+analysts: [max-ostapenko, dooman87]
+translators: [cybai]
+thefoxis_bio: Karolina 是 <a href="https://calibreapp.com/">Calibre</a> 的產品設計主管。她主要負責打造最全面的速度監測平台。她同時也是為您蒐集效能相關訊息與資訊的 <a lang="en" href="https://perf.email/">Performance Newsletter</a> 發起人之一。Karolina 也<a href="https://calibreapp.com/blog/category/web-platform">時常寫有關於效能如何影響使用者體驗的文章</a>。
+discuss: 2045
+results: https://docs.google.com/spreadsheets/d/164FVuCQ7gPhTWUXJl1av5_hBxjncNi0TK8RnNseNPJQ/
+queries: 09_Performance
+featured_quote: 糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障；今年的全球性疫情更凸顯了那些已經存在的屏障。
+featured_stat_1: 25%
+featured_stat_label_1: 擁有 <span lang="en">Good</span> 的 FCP 的手機版網頁
+featured_stat_2: 18%
+featured_stat_label_2: 擁有 <span lang="en">Good</span> 的 TTFB 的手機版網頁
+featured_stat_3: 4%
+featured_stat_label_3: 在 <span lang="en">Lighthouse</span> 6 時效能分數（<span lang="en">Performance Score</span>）未改變的網站
```

Comment:
We really should support `<span lang="en"></span>` in our featured stats, but currently can't :-(
I must look into why not again but in the meantime need to remove these for now unfortunately.

```suggestion
featured_stat_label_1: 擁有 Good 的 FCP 的手機版網頁
featured_stat_2: 18%
featured_stat_label_2: 擁有 Good 的 TTFB 的手機版網頁
featured_stat_3: 4%
featured_stat_label_3: 在 Lighthouse 6 時效能分數（Performance Score）未改變的網站
```

Reply:
Thanks for catching it!